### ISSUE TITLE
Informational: Master branch broken for < 4.1.0, workaround available.

### ISSUE THREAD

**Issue by Olf0** [CODeRUS/github-sfos-build#3]
Commit https://github.com/CODeRUS/github-sfos-build/commit/07d4450db99a50778643a53f2a34362c0580c416, which changed the hardcoded home-directory from `/home/nemo` to `/home/mersdk`, broke [all docker images < 4.1.0](https://hub.docker.com/r/coderus/sailfishos-platform-sdk/tags): `mkdir -p /home/mersdk/build` fails for these with `Permission denied`.

The right solution would be to use either `/home/nemo` or `/home/mersdk` for targeting either "< 4.1.0" or "≥ 4.1.0", but it is questionable if this is worth the effort.

**Comment by CODeRUS** [comment]
does using older action solve issue for old sfos versions? i can set a tag before 07d4450db99a50778643a53f2a34362c0580c416

**Comment by Olf0** [comment]
> does using older action solve issue for old sfos versions?

Yes, I think it would.  But I assume @bionade24 changed the path, because it is then broken for recent SFOS versions, again.

> i can set a tag before [07d4450](https://github.com/CODeRUS/github-sfos-build/commit/07d4450db99a50778643a53f2a34362c0580c416)

IMO just leave it the way it is: This would likely cause more significant breakage.
As denoted: This is primarily informational, to spare others the 30 minutes it took me to analyse this and find the oldest working (and smallest) docker image.

**Comment by bionade24** [comment]
> Yes, I think it would. But I assume @bionade24 changed the path, because it is then broken for recent SFOS versions, again.


If you make 2 jobs for the diffrent SFOS versions, you can use 2 different commits for the sfos-build action. Problem sovled.

**Comment by CODeRUS** [comment]
> > Yes, I think it would. But I assume @bionade24 changed the path, because it is then broken for recent SFOS versions, again.
> 
> If you make 2 jobs for the diffrent SFOS versions, you can use 2 different commits for the sfos-build action. Problem sovled.

thats what i'm talking about, i can push tag for old version for easy access

**Comment by Olf0** [comment]
> thats what i'm talking about, i can push tag for old version for easy access

If you set a tag right after your initial commit https://github.com/CODeRUS/github-sfos-build/commit/30fc33fbedda62ed2e836b0ab4072bd46c6e03ba, that would help the people who have read this and understand that they can then use, for example:
```
uses: coderus/github-sfos-build@old-stable
      with:
        release: ${{ env.sfos_target }}
        arch: ...
```
for [all `sfos_target` < 4.1.0](https://hub.docker.com/r/coderus/sailfishos-platform-sdk/tags).

I would not call this "a solution", but it is a feasible workaround, which is easy to implement.

I would then update the title of this issue and add "workaround available".

**Comment by CODeRUS** [comment]
created old-stable branch and pushed 0.1.0 tag

**Comment by Olf0** [comment]
Closed by PR #6.
